docker-for-mac/space.md

```diff
@@ -4,91 +4,96 @@ keywords: mac, disk
 title: Disk utilization in Docker for Mac
 ---
 
-Docker for Mac stores Linux containers and images in a single, large "disk image" file
-in the Mac filesystem. This is different from Docker on Linux, which usually stores containers
-and images in the `/var/lib/docker` directory.
+Docker Desktop stores Linux containers and images in a single, large "disk image" file in the Mac filesystem. This is different from Docker on Linux, which usually stores containers and images in the `/var/lib/docker` directory.
 
-## Where is the "disk image" file?
+## Where is the disk image file?
 
-To locate the "disk image" file, first select the whale menu icon and then select
-**Preferences...**. When the **Preferences...** window is displayed, select **Disk** and then **Reveal in Finder**:
+To locate the disk image file, select the Docker icon and then
+**Preferences** > **Resources** > **Advanced**.
 
-![Disk preferences](images/settings-disk.png)
+![Disk preferences](images/menu/prefs-advanced.png)
 
-The **Preferences...** window shows how much actual disk space the "disk image" file is consuming.
-In this example, the "disk image" file is consuming 2.4 GB out of a maximum of 64 GB.
-
-Note that other tools might display space usage of the file in terms of the maximum file size, not the actual file size.
+The **Advanced** tab displays the location of the disk image. It also displays the maximum size of the disk image and the actual space the disk image is consuming. Note that other tools might display space usage of the file in terms of the maximum file size, and not the actual file size.
 
 ## If the file is too big
 
-If the file is too big, you can
+If the disk image file is too big, you can:
+
 - move it to a bigger drive,
 - delete unnecessary containers and images, or
 - reduce the maximum allowable size of the file.
 
 ### Move the file to a bigger drive
 
-To move the file, open the **Preferences...** menu, select **Disk**  and then select
-on **Move disk image**. Do not move the file directly in the finder or Docker for Mac will
-lose track of it.
+To move the disk image file to a different location:
 
-### Delete unnecessary containers and images
+1. Select **Preferences** > **Resources** > **Advanced**.
 
-To check whether you have too many unnecessary containers and images:
+2. In the **Disk image location** section, click **Browse** and choose a new location for the disk image.
 
-If your client and daemon API are version 1.25 or later (use the docker version command on the client to check your client and daemon API versions.), you can display detailed space usage information with:
+3. Click **Apply & Restart** for the changes to take effect.
+
+Do not move the file directly in Finder as this can cause Docker Desktop to lose track of the file.
```

Comment:
Is Finder a feature of Mac? Mac users will probably know what this is.

Reply:
Yea, that's right @traci-morrison. Finder is the equivalent of Windows Explorer.